src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
```

Comment:
Using both @Data and individual annotations like @Setter, @Getter, @ToString can lead to unexpected behavior. Choose one approach for consistency.